projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -518,21 +520,13 @@ public String getAnswer(
     return answer;
   }
 
-  /** Return a {@link Container container} contains all testrigs directories inside it. */
+  /**
+   * Returns a {@link Container container} contains all information about the container {@code
+   * containerName}
+   */
   public Container getContainer(String containerName) {
-    return getContainer(getdirContainer(containerName));
-  }
-
-  /** Return a {@link Container container} contains all testrigs directories inside it */
-  public Container getContainer(Path containerDir) {
-    SortedSet<String> testrigs =
-        new TreeSet<>(
-            CommonUtil.getSubdirectories(containerDir.resolve(BfConsts.RELPATH_TESTRIGS_DIR))
-                .stream()
-                .map(dir -> dir.getFileName().toString())
-                .collect(Collectors.toSet()));
-
-    return Container.of(containerDir.toFile().getName(), testrigs);
+    SortedSet<String> testrigs = listTestrigs(containerName);
+    return new Container(containerName, new ArrayList<>(testrigs), new HashMap<>());
```

Comment:
Right, I will modify this to use the storage API call when we have the implementation in storage API, now it is actually a temp function.